release: 2.67.1 (#14928)
canonical/snapd: NEWS.md

> # New in snapd 2.67.1

Reviewer: the interfaces/builtin changes don't seem covered by these

Author: Thanks, I added a comment for each interface stating the additional allowed syscall

# New in snapd 2.67.1
* Fix apparmor permissions to allow snaps access to kernel modules and firmware on UC24, which also fixes the kernel-modules-control interface on UC24
* AppArmor prompting (experimental): disallow /./ and /../ in path patterns
* Fix 'snap run' getent based user lookup in case of bad PATH
* Fix snapd using the incorrect AppArmor version during undo of an refresh for regenerating snap profiles

> * Add new syscalls to base templates

Reviewer 2: ```suggestion
* Add new syscalls to base templates: epoll_pwait2, {get,set,list,remove}xattrat, fchmodat2, fuxtex_* variants, mseal, map_shadow_stack
```

Author: This is better, as discussed agreed the current version is ok, to avoid having to rerun tests.

> # New in snapd 2.67.1

Author: Thanks, fixed

> * Fix kernel-modules-control interface by allowing all snaps access to kernel modules and firmware

Author: Thanks, proposed improvement for Alfonso to review.

> * Fix kernel-modules-control interface by allowing all snaps access to kernel modules and firmware

Author: Thanks, however it is a bit long, what about this shortened version:

"Components: Fix apparmor permissions to allow snaps access to kernel modules and firmware on UC24, which also fixes the kernel-modules-control interface on UC24."

* Fix 'snap run' getent based user lookup in case of bad PATH

> * Fix undo of snapd deb to snap upgrade wrongly using AppArmor from snapd snap to setup profiles

Reviewer 3: ```suggestion
* Fix snapd using the incorrect AppArmor version during undo of an refresh for regenerating snap profiles.
```

Author: Thanks, will improve after gathering all comments.

Author: Done

> * Add missing syscalls to seccomp base template

Reviewer 4: To capture the addition of syscalls aslso to hardware_observe and mount_observe, perhaps something more generic?

```suggestion
- Add missing syscalls to seccomp base template
+Add new syscalls to base templates
```

Author: Thanks, fixed
